google/resource_logging_exclusion.go

```diff
@@ -0,0 +1,84 @@
+package google
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/logging/v2"
+
+	"strings"
+)
+
+func resourceLoggingExclusionSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"name": {
+			Type:     schema.TypeString,
+			Required: true,
+			ForceNew: true,
+		},
+
+		"description": {
+			Type:     schema.TypeString,
+			Optional: true,
+		},
+
+		"filter": {
```

Comment:
Another thing we're trying to be more consistent about these days is sorting our fields as Required -> Optional -> Computed, and alphabetically within each.